rampwf/tests/kits/air_passengers/submissions/starting_kit/submission.py

```diff
@@ -0,0 +1,34 @@
+import os
+import pandas as pd
+from sklearn.preprocessing import FunctionTransformer
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.pipeline import Pipeline
+from sklearn.ensemble import RandomForestRegressor
+
+
+def _merge_weather(X):
```

Comment:
```suggestion
def _merge_external_data(X):
```

The reason is that people will add other data, so let's be open for the name.